pkg/config/config.go

```diff
@@ -65,6 +65,8 @@ type ClientConfig struct {
 	AWSUserProfile string ` yaml:"aws_user_profile"`
 	// AWSUserName is your AWS username
 	AWSUserName *string ` yaml:"aws_username,omitempty"`
+	// OktaProfile is the profile with which to invoke BLESS
+	OktaProfile *string ` yaml:"okta_profile,omitempty"`
```

Comment:
Might be worth making this a struct `*OktaConfig` that then holds things like the MFA,profile, and aws cred ttl